kernel/Kconfig

```diff
@@ -866,6 +866,23 @@ config XIP
 	  supply a linker command file when building your image. Enabling this
 	  option increases both the code and data footprint of the image.
 
+config OFFSET_64BIT
+	bool "Use 64-bit offsets"
```

Comment:
would be great if this went into its own section in Kconfig (menu), i.e. do not put alongside other features and make it obvious is it a type, maybe others will be added to the same menu later.

`OFFSET_64BIT` can be confusing, maybe make it obvious we are talking about a type here: `TYPE_OFFSET_64BIT`.

Reply:
Sure. The closest thing I found to this so far is `TIMEOUT_64BIT`. Should that be moved as well? It also has to do with a type (`k_timeout_t`).